### ISSUE TITLE
Is shoes working on Windows 8.1?

### ISSUE THREAD

**Issue by pietrofxq** [shoes/shoes-deprecated#262]
I just can't use the Packager. The progress bar get stuck in 50% and the program stop responding if I include shoes. If I not include shoes, the package works but the app doesn't open. Am i missing something?

**Comment by ccoupe** [comment]
Packaging hasn't worked on Shoes 3.x for several years. Last I looked, the website it's trying to download from is unresponsive.

**Comment by pietrofxq** [comment]
So what's is being used these days to make an Shoes app turns into an .exe for windows?

**Comment by ccoupe** [comment]
Sadly, nothing - unless you're willing to build Shoes from C source on Windows - then you can include your app.rb  instead of shoes.rb  which isn't the same thing.   I'd like to fix packaging for Shoes 3.2 or 3.3  but it's way down on the priority list. Someone who knows MinGW C on Windows could help a lot. 

Shoes 3.2 @ http://shoes.mvmanila.com/

**Comment by pietrofxq** [comment]
That is sad. Shoes is awesome and if the packager were working it would be perfect. Will wait for the next version or for the updates, then. Thanks

**Comment by ccoupe** [comment]
You can still wrap your script or directory in a .shy and tell the users to download Shoes (.run. .dmg/.bz2, exe) first. A bit of pain for them but that should work on all Shoes versions 3.0 ,3.1, 3.2, & 4 (preview version).  Binary packaging Shoes with a script is wonderfully complex and  horrifyingly complex.  The current crappy code can be made to work for Windows if there was a website to download from.

**Comment by pietrofxq** [comment]
You mean that all I need to make shoes work for windows is upload the .exe to an server? That should be pretty easy... I was thinking about wrapping ruby, shoes and the rb files with a standard packager, and make the installer install each of the programs, but that should be a pain indeed. Shoes is still the best option for creating tools for your personal use, though

**Comment by ccoupe** [comment]
The url of the website is hardcoded into the Shoes ruby code AND in the 'C' launcher code (for windows) so you have to build all of Shoes and any copies of it will be tied to that web server.  It would be your own version of Shoes, different from any other Shoes and that has many implications. It's even more work if you want it to work for OSX and Linux. Work which has to be done first before fixing the packager.

**Comment by ccoupe** [comment]
Shoes 3.2.11 now has packaging for Windows, OSX, and Linux (from any of them, for any of them). Beware. It's a 15 to 20MB thing - it includes Shoes and Ruby with your script.

**Comment by pietrofxq** [comment]
Amazing! Any idea of when you are going to update the website with the installer? :D

**Comment by ccoupe** [comment]
I'm not in charge of the Website. Shoes 3.2 is not a priority for them.   Until then you'll have to download from my website - the packager is also tied to my  website, but that can be changed by regular users if they set up their own Shoes download site (perhaps a large  intranet, e.g.)

**Comment by PragTob** [comment]
We can totally add a download link over at shoesrb.com. Feel free to open an issue/pull request over at https://github.com/shoes/shoesrb.com

**Comment by pietrofxq** [comment]
A pre-compiled version of shoes 3.2 in the official website would be awesome. I'm excited to announce the news to my friends

**Comment by ccoupe** [comment]
I'll open the issue and provide the info. Tobias et all can edit the html. I hate editing html ;^)

**Comment by ccoupe** [comment]
Sometimes the admin needs a reminder to check the pull requests.
